test/index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>QUnit tests for impress.js</title>
+  <link rel="stylesheet" href="qunit/qunit.css">
+</head>
+
+
+
+<body class="impress-not-supported">
+  <!-- HTML elements to be tested with QUnit should typically be inside
+       a qunit-fixture div. Unfortunately that just won't work with impress.js
+       as it wants the div#impress element to be a direct child of body. Or
+       something, I don't know, but it didn't work.
+       
+       So we just have an empty qunit-fixture, and then our impress steps
+       just below it. This just means you have to make sure that each
+       QUnit.test() cleans up after itself, can't rely on QUnit to do it now.
+  -->
+  <div id="qunit-fixture">
+  </div>
+
+  <div id="impress">
+    <div class="step" data-x="-1000" data-y="0">First slide</div>
+    <div class="step" data-x="-800" data-y="0">Second slide</div>
+    <div class="step" data-x="-600" data-y="0">Third slide</div>
+    <div class="step" id="fourth" data-x="-400" data-y="0">Fourth slide</div>
+  </div>
+
+  <div id="qunit"></div>
+
+  <script src="../js/impress.js"></script>
+  <script src="qunit/qunit.js"></script>
+  <script src="helpers.js"></script>
+  <script>
+    // In case tests didn't complete, we are left with a hash/anchor pointing somewhere. But we want to start from scratch:
+    window.location.hash = "";    
+    // I don't understand exactly why, but we must call init() before the actual tests
+    impress().init();
+    
+    QUnit.test( "Initialize Impress.js", function( assert ) {
+      assert.ok( impress, 
+                 "impress declared in global scope" );
```

Comment:
I don't see the value of having the text aligned, after all it is a matter of personal opinion for code style. In my experience, code style discussions tend to take too many resources and start discussions that take forever, that's why it is a common practice in the community to go for consistency instead of discussing about it. That said, if this PR is landed and we choose to go for message alignment, then all tests from now on that uses QUnit should come with the same alignment for consistency. We could even document that style for consultation later.

...

After I wrote the paragraph above I started thinking: If we are having trouble with QUnit since the beginning, doesn't that count as an argument in favor of another more fluent test framework? An assertion like `impress.should.not.be( undefined )` make us not to worry about how to structure the test message and let the framework do it for us. I am definitely playing devil's advocate here, because I like QUnit simplicity.

What do you think, does it makes sense?